### PR TITLE
Localization: Removed placeholder strings in issue #518

### PR DESCRIPTION
**Pull request by HarikalarKutusu**
This PR handles https://github.com/common-voice/sentence-collector/issues/518
+ Some minor changes in en/sentence-collector.ftl

### CODE REVIEW

**Comment by MichaelKohler** [comment]
Thanks for tackling this. The code changes look good to me. The tests are now failing because we need to do https://github.com/common-voice/sentence-collector/issues/496. I will have a look at that tomorrow. I will also test your PR tomorrow.

**Review by MichaelKohler — CHANGES_REQUESTED** [review]
Thanks, I've noticed a bug (see comments) below. Apart from that it looks good functionality-wise as well.

Could you please also run prettier with `npm run format` and commit the changes?

I have set up the test infrastructure so it uses the real localization mechanism. If you want, you can rebase your branch on top of latest main and have a look at the tests. Some tests will still fail due to string changes. Would be great if you could get the tests to pass after that on your branch as well. If you want me to do that, that's fine too.

File: web/src/components/confirm-form.tsx [hunk 1]
```diff
@@ -52,27 +52,19 @@ export default function ConfirmForm(props: Props) {
...
+            <li></li>
           </Localized>
         )}
 
         {unreviewed.length > 0 && (
           <li>
-            <Localized id="sc-confirm-unreviewed" vars={{ countOfUnreviewed: unreviewed.length }}>
-              {`${unreviewed.length} sentences are unreviewed. If you want, you can also review your sentences now before submitting them.`}
-            </Localized>
-            &nbsp;
```
[the remark this answers is not in the document]
Comment (HarikalarKutusu) [reply web/src/components/confirm-form.tsx]: Done...

**Comment by HarikalarKutusu** [comment]
@MichaelKohler I've done the commit but I don't know how to approach your automatic testing routines. I'll be busy in the next ~3 days so if you can handle these it will be much quicker...

**Comment by MichaelKohler** [comment]
You can run `npm test` locally to run all the automated tests. If you do that you will see the following errors:

```
should render loading notice

...Unable to find an element with the text: Loading rejected sentences... 
```

This you changed that to "Loading rejected sentences…", so a simple change in the string we test for is fine:

```
expect(screen.getByText('Loading rejected sentences…')).toBeTruthy();
```

This is the case for both `my-sentences-list.test.tsx` as well as `rejected-sentences.test.tsx`.

```
should render delete error

... Unable to find an element with the text: Failed to delete selected sentences.. Please try again!.
```

Also a changed string. Same solution as above to adjust the text we're testing against.

The `review-criteria.test.tsx` snapshot changed due to the criteria string updates, all changes there look good as well, so updating the test snapshot is the way to go: `npm test -- -u`.

**Review by MichaelKohler — APPROVED** [review]
Thank you!

**Pull request merged**